package.json

```diff
@@ -7,7 +7,7 @@
     "@types/bcrypt": "^5.0.0",
     "@types/cors": "^2.8.12",
     "@types/express": "^4.17.13",
-    "@types/jest": "~26.0.23",
+    "@types/jest": "^27.0.0",
```

Comment:
Is there an automatic tool to identify major version changes? or did you manually have to lookup and change them? For future reference.

Reply:
in npm audit the peer dependencies tell you what the min version needs to be then manually backtrack from there

Reply:
Thanks.